### DISCUSSION TITLE
LGPL license question

### DISCUSSION THREAD

**Discussion by SkyLionx** [FreeOpcUa/opcua-asyncio#1687] [Q&A] [answered]
Hello, for one of my clients I need to use the OPC-UA protocol in order to communicate with an industrial machine.
I'm actually using Python and I have a few questions on the LGPL license.
From what I understood, importing the library in Python is equivalent to dynamically linking and therefore, if the library is used without any modification, the copyleft effect is not triggered.
What happens instead if to distribute it to my client I use some bundling software like PyInstaller or a compiler like nuitka?

**Comment by oroulet** [comment]
My understanding is that this is fine. this is the entire point of having it as LGPL compared to GPL. I am the main author but I am not a lawyer..

**Reply by SkyLionx** [reply, marked as answer]
Thanks for your reply, I'll probably use nuitka and exclude your library to avoid any problem.
